layouts/partials/docs/toc.html

```diff
@@ -65,8 +94,15 @@
                 </div>
             </div>
         {{ end }}
-    </aside>
-{{ end }}
+
+    {{ end }}
+
+    {{ if $isGetStarted }}
+        <h4 class="no-anchor">
+            <a href="{{ relref . "/docs/reference" }}">Reference</a>
+        </h4>
+    {{ end }}
```

Comment:
The changes in this file aren't great -- but it was the simplest way to add these top-level nav elements to the left nav without a more involved change.